README.md

```diff
@@ -76,9 +76,12 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 * Ruby 2.3+
 * Your cluster must be running Kubernetes v1.10.0 or higher<sup>1</sup>
 * Each app must have a deploy directory containing its Kubernetes templates (see [Templates](#using-templates-and-variables))
+* You must remove the` kubectl.kubernetes.io/last-applied-configuration` annotation from any resources in the namespace that you do not wish to be deleted and which are not included in your deploy directory. This annotation is added automatically when you create resources with `kubectl apply`. `kubernetes-deploy` will prune any resources that have this annotation and are not in the deploy directory.<sup>2,3</sup>
```

Comment:
This was deleted in the PR that introduced the ability to manage a subset of the namespace using labels. I don't feel strongly about bringing it back, but it would need to be adjusted to account for the label-based strategy.